python/make_cutflow_table.py

```diff
@@ -126,6 +361,14 @@ def make_table(samples=[], sampleLabels=[], indir = "./cpp/temp_data/", year = "
 samples=bkgsamples+sigsamples
 sampleLabels=bkgsampleLabels+sigsampleLabels
 
-#make_table(samples,sampleLabels)
-make_table(bkgsamples,bkgsampleLabels,extension="_bkg")
-make_table(sigsamples,sigsampleLabels,extension="_sig")
+listofcutflows = []
+listofcutflows = getListOfCutflows()
+toexclude = []
+
+for cutflow in listofcutflows:
+    if cutflow in toexclude:
+        continue
+    make_cutflow_table(cutflow,samples,sampleLabels,doBkgTable=True,doSignalTable=False,doSoverB=False,doSignalOnlyTable=False)
+    make_cutflow_table(cutflow,samples,sampleLabels,doBkgTable=False,doSignalTable=True,doSoverB=False,doSignalOnlyTable=False)
```

Comment:
Maybe it would be worth including the sample as an extension to the signal tables? I understand that they are independent from the bkg ones?

Or even have that extension define the signal sample that we are using, so that we can have more flexibility in the code (changing the extension changes the signal sample to run on)?